code/package.json

```diff
@@ -75,8 +75,8 @@
     "publish:debug": "npm run publish:latest -- --npm-tag=debug --no-push",
     "publish:latest": "lerna publish --exact --concurrency 1 --force-publish",
     "publish:next": "npm run publish:latest -- --npm-tag=next",
-    "storybook:ui": "./lib/cli/bin/index.js dev --port 6006 --config-dir ./ui/.storybook --no-manager-cache",
-    "storybook:ui:build": "./lib/cli/bin/index.js build --config-dir ./ui/.storybook",
+    "storybook:ui": "NODE_OPTIONS=\"--preserve-symlinks --preserve-symlinks-main\" ./lib/cli/bin/index.js dev --port 6006 --config-dir ./ui/.storybook --no-manager-cache",
+    "storybook:ui:build": "NODE_OPTIONS=\"--preserve-symlinks --preserve-symlinks-main\" ./lib/cli/bin/index.js build --config-dir ./ui/.storybook",
```

Comment:
You should probably `cd` into the `ui` directory & run storybook from there. Though I'm not 100% sure it's required.

Reply:
It's actually on purpose we don't do that, because Vite will look for `node_modules` in cwd, and since they're all at `code/node_modules` we have to run from `code`